### ISSUE TITLE
Add cmyk color support to -classify()

### ISSUE THREAD

**Issue by mbloch** [mbloch/mapshaper#488]
* CMYK versions of color ramps
* Support for `cmyk(c,m,y,k)` notation
* Interpolation of cmyk colors

**Comment by Dekkerb** [comment]
If this results in a cmyk-vector file (probably not svg though, or can that be a cmyk file?) which can be opened as CMYK in adobe illustrator or similar -- then this would be awesome for sure. Since RGB colorramps don't translate well to cmyk.

**Comment by mbloch** [comment]
It appears that support for `cmyk()` in CSS and/or SVG is still just at the proposal stage, and not widely implemented. Illustrator ignores cmyk colors when importing from SVG. One option that might work for our newsroom could be to find a way to convert SVG with cmyk colors to PDF files, which could be imported by Illustrator.

**Comment by Dekkerb** [comment]
Yes, that pdf route would be awesome! Also, maybe an Illustrator-script could find the cmyk-values 'associated' with the RGB-colors in the document and swap (remap) them, after converting the ai-file to cmyk? Spoken as someone with zero scripting/coding knowledge, of course.
